source/jsonnet/england-wales/ccs/blocks/individual/ethnic_group_black.jsonnet

```diff
@@ -45,7 +45,7 @@ local question(title, instruction) = {
             id: 'african-ethnic-group-answer-other',
             type: 'TextField',
             mandatory: false,
-            label: 'Enter African background',
+            label: 'Enter African ethic group or background',
```

Comment:
I know the spreadsheet says ethic but surely this should be ethnic right? 😆

Reply:
Whoops, yup literally copied and pasted.